### PR TITLE
add kiwicom/rancher-reaper

### PR DESCRIPTION
**Pull request by hato221**

### CODE REVIEW

File: rancher-reaper/README.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+##Developing
+Suggestions and pull requests welcome at the GitHub repo.
+
+To run locally in Docker, set the environment variables listed above and run:
+
+$ docker-compose up
+Or without docker (local Ruby 2.x installation required):
+
+$ bundle install
+$ bundle exec thin -R lib/config.ru start
```
Comment (underyx) [inline rancher-reaper/README.md]: The formatting of this is completely broken, can you add formatting marks?

File: rancher-reaper/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM ruby:2-alpine
+
+RUN apk update && apk add \
+      alpine-sdk \
+      gettext
+
+WORKDIR /usr/src/app
+COPY Gemfile /usr/src/app/
+COPY Gemfile.lock /usr/src/app/
```
Comment (underyx) [inline rancher-reaper/Dockerfile]: You forgot to add these files

**Comment by underyx** [comment]
What's the point of copying the whole source code? We could also just import the whole image with `FROM ampedandwired/rancher-reaper`.

**Comment by hato221** [comment]
@underyx I will fix what you requested. Main reason for copy&paste is that we use our secrets there, so we would like to build it on our systems.

File: rancher-reaper/README.md (hunk 1 above)
Comment (hato221) [reply rancher-reaper/README.md]: Sorry, yep.

**Comment by aexvir** [comment]
Closing this, we are mainly running on k8s now

**Pull request closed without merging**